dask/dataframe/core.py

```diff
@@ -1356,9 +1381,10 @@ def partitions(self):
         """Slice dataframe by partitions
 
         This allows partitionwise slicing of a Dask Dataframe.  You can perform normal
-        Numpy-style slicing but now rather than slice elements of the array you
+        Numpy-style slicing, including s but now rather than slice elements of the array you
```

Comment:
what is "s" where it says including "s"?

Reply:
Mistake on my part. Fixed.